vanilla-framework/vanilla-brochure-theme
# Muted headings should be left aligned

## Done

Muted headings should be left aligned

## QA

- Run feature branch
- Check http://0.0.0.0:8102/vanilla-brochure-theme/examples/patterns/muted-heading/
- Verify that muted heading is left aligned
